nodejs/eks/cmd/provider/cni.ts

```diff
@@ -71,6 +73,12 @@ function computeVpcCniYaml(cniYamlText: string, args: VpcCniInputs): string {
     if (args.warmIpTarget) {
         env.push({name: "WARM_IP_TARGET", value: args.warmIpTarget.toString()});
     }
+    if (args.warmPrefixTarget) {
+        env.push({name: "WARM_PREFIX_TARGET", value: args.warmPrefixTarget.toString()});
+    }
+    if (args.enablePrefixDelegation) {
+        env.push({name: "ENABLE_PREFIX_DELEGATION ", value: args.enablePrefixDelegation.toString()});
```

Comment:
I think you need to handle the boolean value instead.
```suggestion
        env.push({name: "ENABLE_PREFIX_DELEGATION ", value: args.enablePrefixDelegation ? "true" : "false"});
```